UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/appsettings.json

```diff
@@ -32,6 +32,7 @@
   "KeyVaultSettings": {
     "ServiceUri": ""
   },
+
   "APPINSIGHTS_INSTRUMENTATIONKEY": "",
   "AllowedHosts": "*"
 }
```

Comment:
No change to this file other than space.

Reply:
Removed